### PR TITLE
updates for readability

### PR DESCRIPTION
**Pull request by justin808**
Also, add comment to database.yml

### CODE REVIEW

**Comment by justin808** [comment]
@robwise Check this out.

File: Gemfile [hunk 1]
```diff
@@ -4,9 +4,15 @@ ruby "2.2.2"
 #
 # Bundle edge Rails instead: gem "rails", github: "rails/rails"
 gem "rails", "~>4.2"
-# Use sqlite3 as the database for Active Record
-gem "sqlite3", group: :development
-gem "pg", group: :production
+
+# Note: We're using sqllite3 for development and testing
+gem "sqlite3", group: [:development, :test]
```
Comment (robwise) [inline Gemfile]: Why are we using group blocks for some gems (see line 55, for example) and passing them as hash options for others?

Comment (samnang) [inline Gemfile]: I think maybe no reasons for this, but sometimes the order of loading gems matter.

Comment (justin808) [inline Gemfile]: Whatever makes sense for readability. Usually the DB gems go at the top.

Comment (samnang) [inline Gemfile]: Totally agree with @justin808

**Pull request merged**